integration/tailwind-test.ts

```diff
@@ -358,12 +359,10 @@ test.describe("Tailwind enabled via unstable future flag", () => {
         `,
         "app/root.jsx": js`
           import { Links, Outlet } from "@remix-run/react";
-          import { cssBundleHref } from "@remix-run/css-bundle";
```

Comment:
This is just a clean-up since this fixture doesn't use any CSS bundling features.